atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/SweepQueueReadWriteTest.java

```diff
@@ -90,18 +91,19 @@ public static int putTombstone(KvsSweepQueueWriter writer, long timestamp, Cell
 
     private static int write(KvsSweepQueueWriter writer,
             long timestamp, Cell cell, boolean isTombstone, boolean conservative) {
-        WriteInfo write = WriteInfo.of(conservative ? TABLE_REF : TABLE_REF2, cell, isTombstone, timestamp);
+        TableReference tableRef = conservative ? TABLE_REF : TABLE_REF2;
```

Comment:
You could use `TABLE_CONSERVATIVE` and `TABLE_THOROUGH` here, like in the other test.